packages/discovery-provider/src/tasks/entity_manager/entities/comment.py

```diff
@@ -22,6 +22,12 @@ def validate_comment_tx(params: ManageEntityParameters):
         and comment_id in params.existing_records[EntityType.COMMENT.value]
     ):
         raise IndexingValidationError(f"Comment {comment_id} already exists")
+    if params.metadata["entity_type"] != "Track":
```

Comment:
Yesterday we made a change to just default this instead, I wonder if there's an advantage to throwing an error instead 🤔

Reply:
it matters somewhat because defaulting doesn't protect against someone making up some type. otherwise just hardcoding or removing this column entirely maybe better if we're committing to just tracks.

Reply:
Yeah, makes sense. When I get around to it today I'll update this PR with main and remove the default in favor of this. I'll move the default in the SDK instead